_candidates_en/YoshitakaOkuda_2.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Yoshitaka Okuda
+title: "squbs is awesome"
+length: 10
+audience: Beginner
+language: Japanese
+twitter: https://twitter.com/yoskhdia
+github: https://github.com/yoskhdia
+icon: https://pbs.twimg.com/profile_images/814301502516826112/tkeIfq1H_400x400.jpg
+organization: Supership Inc.
+tags:
+  - Best Practices
+  - Intro to Scala
+  - Software Design and Architecture
+suggestions:
+  - People who want to use or are currently using Akka in production
+  - People who want to focus on feature requirements rather than quality characteristics requirements
+  - People who want to know about squbs
+---
+In order to use Akka in production, we also need to pay attention to logging, monitoring and application life cycle. Although Akka has a rich tool set, it does not cover the above in production level.
+In this session, I will introduce a library called squbs developed by PayPal in order to achieve the goal above.
+squbs's design principal and tools are completely different from all-in-one frameworks. Unfortunately they do not have enough information in Japanese. So, it would be happy for me if this session cloud provide useful information to you.
```

Comment:
+Squbs's design principal and tools are completely different from all-in-one frameworks. Unfortunately, they do not have enough information in Japanese. So, it would make me happy if this session could provide useful information to you.